docs/configure-amazon-role.md

```diff
@@ -35,7 +35,7 @@ Create a new policy for the Service account:
                 "sts:AssumeRole",
                 "sts:GetFederationToken"
             ],
-            "Resource": "arn:aws:iam::123456789:role/*"
+            "Resource": "arn:aws:iam::*:role/*"
         }
```

Comment:
Just to double check, this allows assuming to any role on the service account, so it is a safe change.

Reply:
yup, sorry it should probably be separate, but I really want to add this, because we need the service acc to assume role in any account, not a specific one :)

Reply:
I'll split it off, it really does not belong to this PR :)

Reply:
Splitted into #218